ui/DesignSystem/Component/Remote.svelte

```diff
@@ -1,35 +1,46 @@
-<script>
-  import * as notification from "../../src/notification.ts";
-  import * as remote from "../../src/remote.ts";
+<script lang="ts">
+  import * as notification from "../../src/notification";
+  import * as error from "../../src/error";
+  import * as remote from "../../src/remote";
 
   import WithContext from "./WithContext.svelte";
 
-  export let store = null;
-  export let context = null;
+  export let store: remote.Store<any>;
+  export let context: string | undefined = undefined;
+
+  // Shorthand for casting these states
```

Comment:
Not the prettiest syntax, but a more elegant solution will require a refactor of `remote.ts`, which was built before we could use typescript in components.

Reply:
Can we capture a follow-up issue for that?

Reply:
https://github.com/radicle-dev/radicle-upstream/issues/926